packages/browser/src/Session.ts

```diff
@@ -213,6 +221,8 @@ export class Session extends EventEmitter {
         ) {
           this.info.isLoggedIn = true;
           this.info.webId = reference.webId;
+
+          this.fireSessionRestoreEvent();
```

Comment:
```suggestion
```
With this temporary workaround, we base authentication on the RS cookie, so no redirection happens either, which means there is no need to restore the previous URL.